gax-java/gax-httpjson/src/main/java/com/google/api/gax/httpjson/HttpJsonCallContext.java

```diff
@@ -446,12 +508,20 @@ public HttpJsonCallOptions getCallOptions() {
     return callOptions;
   }
 
+  /** Backport of {@link #getDeadlineInstant()} */
   @Deprecated
   @Nullable
-  public Instant getDeadline() {
+  @ObsoleteApi("Use getDeadlineInstant() instead")
+  public org.threeten.bp.Instant getDeadline() {
     return getCallOptions() != null ? getCallOptions().getDeadline() : null;
   }
 
+  @Deprecated
+  @Nullable
+  public java.time.Instant getDeadlineInstant() {
+    return toJavaTimeInstant(getDeadline());
+  }
```

Comment:
Not sure what our guidance is on this. Should we even introduce an alternative for a method that is already marked as deprecated? Does it make sense to introduce an alternative just to remove it in the next breaking change?

Reply:
I see the point. We probably don't want alternatives for deprecated methods. I'll remove them

Reply:
Removed